pages/public_cloud/public_cloud_network_services/tutorial-ubika_vrack/guide.fr-fr.md

```diff
@@ -0,0 +1,394 @@
+
+#### Configurer le réseau de gestion Ubika WAAP Gateway
+
+> [!primary]
+> Dans ce scénario, nous utiliserons deux machines virtuelles configurées pour l'appliance de sécurité afin d'atteindre la haute disponibilité (HA) et une machine virtuelle supplémentaire pour la gestion. Cette configuration assure une protection du failover et une disponibilité continue du service. Pour plus d'exemples et de conseils détaillés sur les options d'évolutivité, veuillez vous référer à la [documentation Ubika](https://www.ubikasec.com/ressources/){.external}.
+
+Créez un réseau privé pour la gestion de l'infrastructure :
+
+```bash
+openstack network create --provider-network-type vrack --provider-segment 1000 ubika-management
+```
+
+Cette commande crée un réseau privé pour gérer les instances UWG, en utilisant le réseau privé virtuel (vRack) fourni par OVHcloud. Ce réseau isolé est destiné à la communication interne entre les composants d'Ubika.
+
+```bash
+openstack subnet create --network ubika-management --subnet-range 192.168.1.0/24 --dhcp --gateway none --dns-nameserver 213.186.33.99 ubika-management
+```
+
+Configurez ici un sous-réseau pour le réseau de gestion, en spécifiant une plage d'adresses IP et un serveur DNS pour les communications internes.
+
+#### Configurer le réseau de charge de travail Ubika WAAP Gateway
+
+Créez un réseau privé pour la charge de travail :
+
+```bash
+openstack network create --provider-network-type vrack --provider-segment 0 --disable-port-security ubika-workload
+```
+
+Cette commande crée un réseau privé pour la charge de travail, conçu pour héberger des applications sécurisées par UWG.
+
+```bash
+openstack subnet create --network ubika-workload --subnet-range 192.168.2.0/24 --dhcp --dns-nameserver 213.186.33.99 ubika-workload
+```
+
+Définissez ici un sous-réseau pour le réseau de charge de travail, ce qui permet une gestion efficace du trafic réseau.
+
+Créez une passerelle :
+
+> [!primary]
+> Une passerelle peut être nécessaire pour accéder à Internet à partir de serveurs web, notamment pour des tâches telles que l'installation de logiciels (Nginx par exemple) ou la gestion à distance. Toutefois, pour le trafic client entrant, ce composant n'est pas utilisé.
+
+```bash
+openstack router create --external-gateway Ext-Net ubika-workload
+```
+
+```bash
+openstack router add subnet 2481bcaf-efa2-419a-ad92-d6d27737dfd1 ubika-workload
+```
+
+#### Déployer les instances Ubika WAAP Gateway
+
+Téléversez l'image UWG image vers OpenStack:
+
+Rendez-vous dans la section `download`{.action} du [site officiel d'Ubika](https://my.ubikasec.com/){.external}. Connectez-vous à votre compte Ubika et suivez les instructions pour télécharger l'image UWG OpenStack.
+
+Rendez-vous dans le dossier où vous avez téléchargé votre image UWG OpenStack et téléversez l'image UWG OpenStack (pour ce tutoriel, nous utilisons l'image `UBIKA_WAAP_Gateway-generic-cloud-6.11.10+51a56f6201.b56855.qcow2`) :
```

Comment:
I chose to translate closely the "download/upload" terms from the EN version. But this leads to this step now looking a bit weird. Can you check if this is appropriate? (EN/FR)

Reply:
for me the translation is okay